diff --git a/dist/index.d.ts b/dist/index.d.ts
index 281c9faf8930b67cbfd86dfc4932ea221872671c..3877600701c3685669aa44ce5472cc2e8cc0b435 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,7 +1,7 @@
-import { Env } from "markdown-it";
-import { PluginWithOptions } from "@mdit/helper";
+import type { MarkdownExitEnv } from "markdown-exit";
+import type { PluginWithOptions } from "@mdit/helper";
 //#region src/options.d.ts
-interface MarkdownItTexOptions<MarkdownItEnv = Env> {
+interface MarkdownItTexOptions<MarkdownItEnv = MarkdownExitEnv> {
   /**
    * Math delimiter syntax to enable
    *
