diff --git a/lib/index.d.ts b/lib/index.d.ts
index 1d23b74661b8c741c8c05ea174535f87e85768b2..3924d91a151139008b641998ff194a6a45460b57 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,5 +1,4 @@
-import { RenderRule } from 'markdown-it/lib/renderer.mjs';
-import { PluginWithOptions } from 'markdown-it';
+import type { PluginWithOptions, RenderRule } from 'markdown-exit';
 
 interface MarkdownItAlertOptions {
     /**
