diff --git a/dist/index.d.ts b/dist/index.d.ts
index 31812788f7b876d6db9aaa07ba5153493fe9047d..5b8e5e5f42ab23c0cc4e398929071acac6756e5a 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from "markdown-it";
-import Token from "markdown-it/lib/token.mjs";
+import type { PluginSimple, Token } from "markdown-exit";
 
 //#region src/plugin.d.ts
 declare const footnote: PluginSimple;
