diff --git a/lib/index.d.ts b/lib/index.d.ts
index a719e7f4041f6ebc763a28298286e9a87016ddb3..3127e11d6833875baea87ed5fb944c4e56569c51 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from "markdown-it";
-import Token from "markdown-it/lib/token.mjs";
+import type { PluginSimple, Token } from 'markdown-exit';
 
 //#region src/plugin.d.ts
 declare const footnote: PluginSimple;
