diff --git a/dist/index.d.ts b/dist/index.d.ts
index c51895f2aa910f42cce8464fe4a8eddfd93e67a3..2497ec79c04e063fcaa197e8e45955e855cc78f9 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,4 +1,4 @@
-import { InlineRule, PluginSimple } from "@mdit/helper";
+import type { InlineRule, PluginSimple } from "@mdit/helper";
 //#region src/legacy.d.ts
 /**
  * @deprecated Recommended to use `imgSize` instead.
