diff --git a/dist/index.d.ts b/dist/index.d.ts
index 9f53be52b1ea79cc1870834ee26490ec1e77950f..bccf1fa501d471a36c951748298f2291017f2914 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from "markdown-it";
-import { RuleInline } from "markdown-it/lib/parser_inline.mjs";
+import type { PluginSimple, RuleInline } from "markdown-exit";
 //#region src/legacy.d.ts
 /**
  * @deprecated Recommended to use `imgSize` instead.
