diff --git a/dist/index.d.ts b/dist/index.d.ts
index e3237e82053a1382a5c703a7d0a0b08f6eb80a05..2f87fd87a9ba8c46eeea53d87efa6eb778ef242b 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,5 @@
-import { PluginSimple } from "@mdit/helper";
-import { Env, Token } from "markdown-it";
+import type { PluginSimple } from "@mdit/helper";
+import type { MarkdownExitEnv, Token } from "markdown-exit";
 //#region src/plugin.d.ts
 export declare const footnote: PluginSimple;
 //#endregion
@@ -11,7 +11,7 @@ interface FootNoteToken extends Token {
     label: string;
   };
 }
-interface FootNoteEnv extends Env {
+interface FootNoteEnv extends MarkdownExitEnv {
   docId?: string;
   footnotes?: {
     label?: string;
