diff --git a/dist/index.d.ts b/dist/index.d.ts
index 929262f1d645fdf575a2032e94c7f08827987570..7c17eb1cef25980fa1e5b626cba37e3eee183fc6 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,6 +1,6 @@
-import { PluginWithOptions } from "markdown-it";
+import type { PluginWithOptions, MarkdownExitEnv } from "markdown-exit";
 //#region src/options.d.ts
-interface MarkdownItTexOptions<MarkdownItEnv = unknown> {
+interface MarkdownItTexOptions<MarkdownExitEnv = {}> {
   /**
    * Math delimiter syntax to enable
    *
@@ -38,7 +38,7 @@ interface MarkdownItTexOptions<MarkdownItEnv = unknown> {
    * Tex 渲染函数
    * @returns 渲染结果
    */
-  render: (content: string, displayMode: boolean, env: MarkdownItEnv) => string;
+  render: (content: string, displayMode: boolean, env: MarkdownExitEnv) => string;
   /**
    * Whether to allow inline math with spaces on ends
    *
