diff --git a/dist/index.d.ts b/dist/index.d.ts
index bcae32482dad54b555ce0b032c680bc0088b4b92..582add257e3ae19de1f35e0e47b4f32653c08ac4 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,5 @@
-import { PluginWithOptions } from "@mdit/helper";
-import { Env } from "markdown-it";
+import type { PluginWithOptions } from "@mdit/helper";
+import type { MarkdownExitEnv } from "markdown-exit";
 //#region src/options.d.ts
 interface MarkdownItTaskListOptions {
   /**
@@ -57,7 +57,7 @@ export declare const tasklist: PluginWithOptions<MarkdownItTaskListOptions>;
 //#endregion
 //#region src/types.d.ts
 declare const tasklistIdKey: unique symbol;
-interface TaskListEnv extends Env {
+interface TaskListEnv extends MarkdownExitEnv {
   [tasklistIdKey]?: number;
 }
 //#endregion
