diff --git a/lib/index.d.ts b/lib/index.d.ts
index 10c66df256628798b4f2207572fe99454d192feb..69dd04c5d6e7f4d1c2e326d46cff3b379728df37 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from 'markdown-it';
-import Token from 'markdown-it/lib/token.mjs';
+import type { PluginSimple, Token } from 'markdown-exit';
 
 /**
  * Forked from https://github.com/markdown-it/markdown-it-footnote/blob/master/index.mjs
