diff --git a/lib/index.d.ts b/lib/index.d.ts
index 7c4b8f191022e2c9acd42aaa3fdcede34a216319..11b9e8f5baeec1bcf0ffefe919434246d014745d 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,6 +1,4 @@
-import { Options, PluginWithOptions } from 'markdown-it';
-import Renderer, { RenderRule } from 'markdown-it/lib/renderer.mjs';
-import Token from 'markdown-it/lib/token.mjs';
+import type { MarkdownExitEnv, MarkdownExitOptions, PluginWithOptions, Renderer, RenderRule, Token } from 'markdown-exit';
 
 interface MarkdownItTabData {
     /**
@@ -56,7 +54,7 @@ interface MarkdownItTabOptions {
     /**
      * Tabs open render
      */
-    openRender?: (info: MarkdownItTabInfo, tokens: Token[], index: number, options: Options, env: any, self: Renderer) => string;
+    openRender?: (info: MarkdownItTabInfo, tokens: Token[], index: number, options: MarkdownExitOptions, env: MarkdownExitEnv, self: Renderer) => string;
     /**
      * Tabs close render
      */
@@ -64,7 +62,7 @@ interface MarkdownItTabOptions {
     /**
      * tab open render
      */
-    tabOpenRender?: (data: MarkdownItTabData, tokens: Token[], index: number, options: Options, env: any, self: Renderer) => string;
+    tabOpenRender?: (data: MarkdownItTabData, tokens: Token[], index: number, options: MarkdownExitOptions, env: MarkdownExitEnv, self: Renderer) => string;
     /**
      * tab close render
      */
