diff --git a/lib/index.d.ts b/lib/index.d.ts
index 96f20dfacd9b204681f64c85b7a2499f384c060f..f66db374d448a84186b302782c8a174051a2ca9e 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,6 +1,6 @@
-import { PluginWithOptions } from 'markdown-it';
+import type { PluginWithOptions, MarkdownExitEnv } from 'markdown-exit';
 
-interface MarkdownItTexOptions<MarkdownItEnv = unknown> {
+interface MarkdownItTexOptions<MarkdownExitEnv = {}> {
     /**
      * Math delimiter syntax to enable
      *
@@ -40,7 +40,7 @@ interface MarkdownItTexOptions<MarkdownItEnv = unknown> {
      * @param env MarkdownIt 环境变量
      * @returns 渲染结果
      */
-    render: (content: string, displayMode: boolean, env: MarkdownItEnv) => string;
+    render: (content: string, displayMode: boolean, env: MarkdownExitEnv) => string;
     /**
      * Whether to allow inline math with spaces on ends
      *
