diff --git a/dist/index.d.ts b/dist/index.d.ts
index 17319eb0454d862deab324b25b964051ecfc20c8..87715f937e92e8a86cc7c4b54d56075453c13e55 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,4 @@
-import { RenderRule } from "markdown-it/lib/renderer.mjs";
-import { PluginWithOptions } from "markdown-it";
+import type { PluginWithOptions, RenderRule } from "markdown-exit";
 
 //#region src/options.d.ts
 interface MarkdownItAlertOptions {
