diff --git a/dist/index.d.ts b/dist/index.d.ts
index 75fc1b6e339cf2f20d65529bcffc5dc516cdd2c4..0179d4b91586066cca0f7f83bde0bb3ab276eaec 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from "markdown-it";
-import { RuleInline } from "markdown-it/lib/parser_inline.mjs";
+import type { PluginSimple, RuleInline } from "markdown-exit";
 
 //#region src/legacy.d.ts
 /**
