diff --git a/dist/index.d.ts b/dist/index.d.ts
index 59d0d6ada5d2a8bb5a0805593ca4c85f4e31523e..606dcedb198ba75df8eca74b2ccdc5682cb516a9 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from "markdown-it";
-import Token from "markdown-it/lib/token.mjs";
+import type { PluginSimple, Token } from "markdown-exit";
 //#region src/plugin.d.ts
 declare const footnote: PluginSimple;
 //#endregion
