diff --git a/lib/index.d.ts b/lib/index.d.ts
index e1eb433bce6292f5fd1ecdffcf4080eb2c54c980..7132ab76ea38f3423112b4934be96c78ebb584d3 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,5 +1,4 @@
-import { PluginSimple } from "markdown-it";
-import { RuleInline } from "markdown-it/lib/parser_inline.mjs";
+import type { PluginSimple, RuleInline } from 'markdown-exit';
 
 //#region src/legacy.d.ts
 /**
