diff --git a/dist/index.d.ts b/dist/index.d.ts
index b9c48778e3bb9f873505e0a0b344eb878a4a6361..99d61c2b3ad080bd9abd1d271aebe70d139152e3 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,5 +1,5 @@
-import { RendererRule } from "markdown-it";
-import { PluginWithOptions } from "@mdit/helper";
+import type { RendererRule } from "markdown-exit";
+import type { PluginWithOptions } from "@mdit/helper";
 //#region src/options.d.ts
 interface MarkdownItAlertOptions {
   /**
