diff --git a/dist/index.d.ts b/dist/index.d.ts
index addb1598c0101c6c25f605f1d2c5740e0e482d0a..bcb2cc749c5d99325463c3f4f429b78668d29c43 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,4 +1,4 @@
-import { PluginWithOptions } from "@mdit/helper";
+import type { PluginWithOptions } from "@mdit/helper";
 //#region src/options.d.ts
 interface MarkdownItSpoilerOptions {
   /** @default "span" */
