diff --git a/dist/index.d.ts b/dist/index.d.ts
index b134d9b357e0e9d639226abeb22a86bbbaf6c0b9..12f456af4d792f3e0725027d2e7c6abf364e4be2 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,6 +1,4 @@
-import { Options, PluginWithOptions } from "markdown-it";
-import Renderer, { RenderRule } from "markdown-it/lib/renderer.mjs";
-import Token from "markdown-it/lib/token.mjs";
+import type { MarkdownExitEnv, MarkdownExitOptions, PluginWithOptions, Renderer, RenderRule, Token } from "markdown-exit";
 
 //#region src/options.d.ts
 interface MarkdownItTabData {
@@ -47,8 +45,8 @@ interface MarkdownItTabInfo {
    */
   data: MarkdownItTabData[];
 }
-type TabsOpenRender = (info: MarkdownItTabInfo, tokens: Token[], index: number, options: Options, env: any, self: Renderer) => string;
-type TabOpenRender = (data: MarkdownItTabData, tokens: Token[], index: number, options: Options, env: any, self: Renderer) => string;
+type TabsOpenRender = (info: MarkdownItTabInfo, tokens: Token[], index: number, options: MarkdownExitOptions, env: MarkdownExitEnv, self: Renderer) => string;
+type TabOpenRender = (data: MarkdownItTabData, tokens: Token[], index: number, options: MarkdownExitOptions, env: MarkdownExitEnv, self: Renderer) => string;
 interface MarkdownItTabOptions {
   /**
    * The name of the tab container.
