diff --git a/lib/index.d.ts b/lib/index.d.ts
index f39badbc4e5cdc04ef24512ee15925ade7ae2944..4c107325165c5a4ee7d9676f158af3b21f9ff661 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,5 +1,4 @@
-import { RenderRule } from "markdown-it/lib/renderer.mjs";
-import { PluginWithOptions } from "markdown-it";
+import type { PluginWithOptions, RenderRule } from 'markdown-exit';
 
 //#region src/options.d.ts
 interface MarkdownItAlertOptions {
