diff --git a/dist/index.d.ts b/dist/index.d.ts
index c4617ba44492907c0ddf896c557ff81098eee89f..221a82fbc94546e04705bef2d6d5233165a38bbe 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1116,4 +1116,4 @@ declare function createMarkdownExit(presetName: PresetName, options?: MarkdownEx
 type MarkdownExitConstructor = InstanceType<typeof MarkdownExit>;
 declare const MarkdownExitConstructor: (typeof createMarkdownExit & typeof MarkdownExit);
 //#endregion
-export { HTMLAttribute, MarkdownExit, MarkdownExitOptions, Nesting, PluginSimple, PluginWithOptions, PluginWithParams, PresetName, RenderOptions, RenderRule, RenderRuleRecord, Renderer, type RuleBlock, type RuleCore, type RuleInline, RuleOptions, Ruler, SourceMapLineRange, Token, createMarkdownExit, MarkdownExitConstructor as default };
\ No newline at end of file
+export { HTMLAttribute, MarkdownExit, MarkdownExitEnv, MarkdownExitOptions, Nesting, PluginSimple, PluginWithOptions, PluginWithParams, PresetName, RenderOptions, RenderRule, RenderRuleRecord, Renderer, type RuleBlock, type RuleCore, type RuleInline, RuleOptions, Ruler, SourceMapLineRange, Token, type StateCore, createMarkdownExit, MarkdownExitConstructor as default };
