diff --git a/dist/index.d.ts b/dist/index.d.ts
index 9dddb34a9b4e1e3aec3ca2592c6de0e541146bcd..fb738ea137f8b464110de40732dba8e4a30b8ccd 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1130,4 +1130,4 @@ declare function createMarkdownExit(presetName: PresetName, options?: MarkdownEx
 type MarkdownExitConstructor = InstanceType<typeof MarkdownExit>;
 declare const MarkdownExitConstructor: (typeof createMarkdownExit & typeof MarkdownExit);
 //#endregion
-export { HTMLAttribute, MarkdownExit, MarkdownExitOptions, Nesting, Parser, ParserOptions, PluginSimple, PluginWithOptions, PluginWithParams, PresetName, RenderOptions, RenderRule, RenderRuleRecord, Renderer, type RuleBlock, type RuleCore, type RuleInline, RuleOptions, Ruler, SourceMapLineRange, StateBlock, StateCore, StateInline, Token, createMarkdownExit, MarkdownExitConstructor as default, defaultOptions };
\ No newline at end of file
+export { HTMLAttribute, MarkdownExit, MarkdownExitEnv, MarkdownExitOptions, Nesting, Parser, ParserOptions, PluginSimple, PluginWithOptions, PluginWithParams, PresetName, RenderOptions, RenderRule, RenderRule as RendererRule, RenderRuleRecord, Renderer, type RuleBlock, type RuleCore, type RuleInline, RuleOptions, Ruler, SourceMapLineRange, StateBlock, StateCore, StateInline, Token, createMarkdownExit, MarkdownExitConstructor as default, defaultOptions };
