diff --git a/dist/core.js b/dist/core.js
index 09f7ac94f41eb9a5b23c96dfb39be8c2f2666c74..19888acc306c8d670fbb7c0e2417bba481535792 100644
--- a/dist/core.js
+++ b/dist/core.js
@@ -65,8 +65,9 @@ function highlightAuto(md, hljs, ignoreIllegals, code, lang) {
   }
 }
 function wrapCodeRenderer(renderer) {
-  return function wrappedRenderer(...args) {
-    return renderer(...args).replace(/<code class="/g, '<code class="hljs ').replace(/<code>/g, '<code class="hljs">');
+  return async function wrappedRenderer(...args) {
+    const html = await renderer(...args);
+    return html.replace(/<code class="/g, '<code class="hljs ').replace(/<code>/g, '<code class="hljs">');
   };
 }
 function inlineCodeLanguageRule(state) {
@@ -99,13 +100,13 @@ function inlineCodeLanguageRule(state) {
     }
   }
 }
-function inlineCodeRenderer(tokens, idx, options, env, slf) {
+function inlineCodeRenderer(tokens, idx, options, env = {}, slf) {
   var _a, _b;
   const token = tokens[idx];
   if (options.highlight == null) {
     throw new Error("`options.highlight` was null, this is not supposed to happen");
   }
-  const highlighted = options.highlight(token.content, (_b = (_a = token.meta) == null ? void 0 : _a.highlightLanguage) != null ? _b : "", "");
+  const highlighted = options.highlight(token.content, (_b = (_a = token.meta) == null ? void 0 : _a.highlightLanguage) != null ? _b : "", "", env);
   return `<code${slf.renderAttrs(token)}>${highlighted}</code>`;
 }
 function core(md, opts) {
diff --git a/src/core.ts b/src/core.ts
index ae09f4bad59be726e6921dfe3f0e6c1797a56feb..96493b37ebaf1ff006340329ed609b79ea8d880e 100644
--- a/src/core.ts
+++ b/src/core.ts
@@ -1,7 +1,12 @@
-import MarkdownIt from 'markdown-it'
-import Renderer from 'markdown-it/lib/renderer'
-import StateCore from 'markdown-it/lib/rules_core/state_core'
-import Token from 'markdown-it/lib/token'
+import type {
+  MarkdownExit,
+  MarkdownExitEnv,
+  MarkdownExitOptions,
+  Renderer,
+  RenderRule,
+  StateCore,
+  Token,
+} from 'markdown-exit'
 import { HLJSApi, LanguageFn } from 'highlight.js'
 
 export interface HighlightOptions {
@@ -62,7 +67,7 @@ function registerLangAliases (hljs: HLJSApi, register: { [lang: string]: string
 }
 
 // Highlight with given language.
-function highlight (md: MarkdownIt, hljs: HLJSApi, ignoreIllegals: boolean, code: string, lang: string): string {
+function highlight (md: MarkdownExit, hljs: HLJSApi, ignoreIllegals: boolean, code: string, lang: string): string {
   try {
     return hljs.highlight(code, { language: lang !== '' ? lang : 'plaintext', ignoreIllegals }).value
   } catch (e) {
@@ -71,7 +76,7 @@ function highlight (md: MarkdownIt, hljs: HLJSApi, ignoreIllegals: boolean, code
 }
 
 // Highlight with given language or automatically.
-function highlightAuto (md: MarkdownIt, hljs: HLJSApi, ignoreIllegals: boolean, code: string, lang: string): string {
+function highlightAuto (md: MarkdownExit, hljs: HLJSApi, ignoreIllegals: boolean, code: string, lang: string): string {
   if (lang !== '') {
     return highlight(md, hljs, ignoreIllegals, code, lang)
   }
@@ -83,9 +88,10 @@ function highlightAuto (md: MarkdownIt, hljs: HLJSApi, ignoreIllegals: boolean,
   }
 }
 // Wrap a render function to add `hljs` class to code blocks.
-function wrapCodeRenderer (renderer: Renderer.RenderRule): Renderer.RenderRule {
-  return function wrappedRenderer (...args) {
-    return renderer(...args)
+function wrapCodeRenderer (renderer: RenderRule): RenderRule {
+  return async function wrappedRenderer (...args) {
+    const html = await renderer(...args)
+    return html
       .replace(/<code class="/g, '<code class="hljs ')
       .replace(/<code>/g, '<code class="hljs">')
   }
@@ -133,7 +139,7 @@ function inlineCodeLanguageRule (state: StateCore): void {
   }
 }
 
-function inlineCodeRenderer (tokens: Token[], idx: number, options: MarkdownIt.Options, env: any, slf: Renderer): string {
+function inlineCodeRenderer (tokens: Token[], idx: number, options: MarkdownExitOptions, env: MarkdownExitEnv = {}, slf: Renderer): string {
   const token = tokens[idx]
 
   // Make TypeScript happy...
@@ -141,12 +147,12 @@ function inlineCodeRenderer (tokens: Token[], idx: number, options: MarkdownIt.O
     throw new Error('`options.highlight` was null, this is not supposed to happen')
   }
 
-  const highlighted = options.highlight(token.content, token.meta?.highlightLanguage ?? '', '')
+  const highlighted = options.highlight(token.content, token.meta?.highlightLanguage ?? '', '', env)
 
   return `<code${slf.renderAttrs(token)}>${highlighted}</code>`
 }
 
-export default function core (md: MarkdownIt, opts?: HighlightOptions): void {
+export default function core (md: MarkdownExit, opts?: HighlightOptions): void {
   const optsWithDefaults = { ...core.defaults, ...opts }
 
   if (optsWithDefaults.hljs == null) {
diff --git a/src/index.ts b/src/index.ts
index 32f83bb95e1e5e089bc13f299934d70fbf93832f..886e1cf79bfd3152db23ae6d1c7118ac6cc010c9 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -1,8 +1,8 @@
-import MarkdownIt from 'markdown-it'
+import type { MarkdownExit } from 'markdown-exit'
 import hljs from 'highlight.js'
 import core, { HighlightOptions } from './core.js'
 
-export default function highlightjs (md: MarkdownIt, opts?: HighlightOptions): void {
+export default function highlightjs (md: MarkdownExit, opts?: HighlightOptions): void {
   opts = { ...highlightjs.defaults, ...opts }
 
   if (opts.hljs == null) {
diff --git a/types/core.d.ts b/types/core.d.ts
index bbc0a3c487b7f2698e5db0b33f23d7c671e1aaa8..14369ff4a5f6a6d9039b7a5963297adce397dc3f 100644
--- a/types/core.d.ts
+++ b/types/core.d.ts
@@ -1,4 +1,4 @@
-import MarkdownIt from 'markdown-it';
+import type { MarkdownExit } from 'markdown-exit';
 import { HLJSApi, LanguageFn } from 'highlight.js';
 export interface HighlightOptions {
     /**
@@ -36,7 +36,7 @@ export interface HighlightOptions {
      */
     ignoreIllegals?: boolean;
 }
-declare function core(md: MarkdownIt, opts?: HighlightOptions): void;
+declare function core(md: MarkdownExit, opts?: HighlightOptions): void;
 declare namespace core {
     var defaults: {
         auto: boolean;
diff --git a/types/index.d.ts b/types/index.d.ts
index 2c8367e72033cb840111cf303b928b73168d2d77..133316c4d35b1e0ea1fbb61dac73fc1017b0e043 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,6 +1,6 @@
-import MarkdownIt from 'markdown-it';
+import type { MarkdownExit } from 'markdown-exit';
 import { HighlightOptions } from './core.js';
-declare function highlightjs(md: MarkdownIt, opts?: HighlightOptions): void;
+declare function highlightjs(md: MarkdownExit, opts?: HighlightOptions): void;
 declare namespace highlightjs {
     var defaults: {
         auto: boolean;
