diff --git a/index.d.ts b/index.d.ts
index 90ed51e4ec640dd968e5820994ab0d8dd2510ca4..14227a4ce03c2f29304bd80bed31e53be433c050 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,4 +1,4 @@
-import MarkdownIt = require('markdown-it');
+import type { MarkdownExit } from "markdown-exit";
 
 declare namespace markdownItAttrs {
   type AllowedAttribute = string | RegExp;
@@ -13,6 +13,6 @@ declare namespace markdownItAttrs {
   }
 }
 
-declare function markdownItAttrs(md: MarkdownIt, options?: markdownItAttrs.Options): void;
+declare function markdownItAttrs(md: MarkdownExit, options?: markdownItAttrs.Options): void;
 
 export = markdownItAttrs;
